torch/export/_trace.py

```diff
@@ -204,6 +204,10 @@ def _convert_input_to_fake(gm, args, kwargs):
             fake_val = node.meta["val"]
             if fake_val is not None and isinstance(fake_val, torch.Tensor):
                 fake_inps.append(fake_val)
+        elif "example_value" in node.meta:
```

Comment:
we should rename fake_inps to something like, fake_vals, since these fake values are not necessarily only from inputs. Also, we don't need to go down this path if there are inputs (we only need to go down this path if there are no outputs), so we can just break if `len(fake_inps) > 0`.

Reply:
fixed now! I didn't break from the loop because `fake_inp` is used later to map `fake_args`. I added a condition to check "example_value" only if  `len(fake_inp)==0`.